content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -39,12 +39,10 @@ localization.
      - Adds the *Propuesta F29* and *Balance Tributario (8 columnas)* reports.
    * - :guilabel:`Chile - E-invoicing`
      - `l10n_cl_edi`
-     - Includes all technical and functional requirements to generate and receive electronic
-       invoices online based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.
-   * - :guilabel:`Chile - Electronic Receipt`
-     - `l10n_cl_boletas`
-     - Includes all technical and functional requirements to generate and receive electronic
-       invoices digitally based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.
+     - Includes all technical and functional requirements to receive and generate electronic
+       receipts and invoices online based on the :abbr:`SII (Servicio de Impuestos Internos)`
+       regulations. Previous versions had two different modules, now it is merged into this main
+       one, required for EDI workflows.
```

Comment:
Hi @gmz-odoo and @dade-odoo, thanks for the feedback and context! 

In my opinion I think it would be best to re-target this PR onto saas-17.4 to accommodate the specific changes from this version, and FWP 18.0 (note the "receipts" word is included):
```suggestion
     - Includes all technical and functional requirements to receive and generate electronic
       receipts and invoices online based on the :abbr:`SII (Servicio de Impuestos Internos)`
       regulations.
```

For the versions 16.0 & 17.0, it does seem like an update is needed for the `Chile - Electronic Receipt` description to change "invoices" to "receipts", specifically:

> **Chile - Electronic Receipt**
> `l10n_cl_boletas`
> Includes all technical and functional requirements to generate and receive electronic ~~invoices~~ receipts digitally based on the :abbr:`SII (Servicio de Impuestos Internos)` regulations.

@gmz-odoo does this sound correct? If so, I can open another PR on 16.0 and FWP this change up to saas-17.2.

Reply:
Hi @samueljlieber , wonderful!

Agreed:
1) For 17.4+ it should be clear that is only one module, and we specify it covers receipt and invoices
- Let me know if there is something needed from my end on this topic

2) For 16.0 to 17.2 we could update that description using 'receipt' on the line for l10n_cl_boletas

I'm around, thanks a lot. 
٩(◕‿◕)

Reply:
Ah yes, just now seeing that you all resolved this after I wrote out my last comment haha